dadb/src/main/kotlin/dadb/AdbKeyPair.kt

```diff
@@ -100,6 +100,9 @@ class AdbKeyPair(
                 it.genKeyPair()
             }
 
+            privateKeyFile.absoluteFile.parentFile?.mkdirs()
+            publicKeyFile.absoluteFile.parentFile?.mkdirs()
```

Comment:
This is to generate .android folder if it doesn't exist?

Reply:
Yeah exactly

Reply:
Got it, don't we need to pass the name of directory as .android?

Reply:
This is a generic method that can generate the key to any location. This logic ensure that no matter where you point to, the parent directories will be created if they don't already exist

Reply:
Ahhh gotcha!